src/Cache/SchemaCache.php

```diff
@@ -91,6 +96,11 @@ public function isEnabled(): bool
         return $this->enabled;
     }
 
+    public function has($key): bool
```

Comment:
That's very low level. Is it possible to hide it?

Reply:
Remove done.